parlai/core/torch_ranker_agent.py

```diff
@@ -224,11 +225,15 @@ def _build_candidates(self, batch, source, mode):
                     "".format(m='candidates' if mode == 'train' else 'eval-candidates'))
 
             cands = batch.candidates
-            cand_vecs = padded_3d(batch.candidate_vecs, use_cuda=self.use_cuda)
+            cand_vecs = padded_3d(batch.candidate_vecs, self.NULL_IDX,
+                                  use_cuda=self.use_cuda)
             if label_vecs is not None:
                 label_inds = label_vecs.new_empty((batchsize))
                 for i, label_vec in enumerate(label_vecs):
-                    label_vec_pad = label_vec.new_zeros(cand_vecs[i].size(1))
+                    label_vec_pad = label_vec.new_zeros(cand_vecs[i].size(1)) \
```

Comment:
I think elsewhere in the code we usually use parenthesis to group multi-line statements rather than backslashes.